test/runtime/kafka.go

```diff
@@ -134,11 +137,12 @@ var _ = Describe("RuntimeKafka", func() {
 	})
 
 	JustBeforeEach(func() {
+		testStartTime = time.Now()
 		monitorRes, monitorStop = vm.MonitorStart("--type l7")
 	})
 
 	JustAfterEach(func() {
-		vm.ValidateNoErrorsInLogs(CurrentGinkgoTestDescription().Duration)
+		vm.ValidateNoErrorsInLogs(time.Since(testStartTime))
```

Comment:
Could you send this fix as a separate pull request so we can backport it? :pray:

Reply:
Yes will do!

Reply:
https://github.com/cilium/cilium/pull/17351